shell/platform/fuchsia/dart_runner/dart_runner.h

```diff
@@ -33,6 +34,10 @@ class DartRunner : public fuchsia::sys::Runner,
       fidl::InterfaceRequest<fuchsia::component::runner::ComponentController>
           controller) override;
 
+  std::map<DartTestComponentControllerV2*,
+           std::shared_ptr<DartTestComponentControllerV2>>
+      test_components_;
```

Comment:
Since this isn't directly read for any logic but is just used to keep our components alive in memory until they're done, maybe we should add some documentation here explaining that.